### ISSUE TITLE
If I get the bbox and label of an image with other detector, how do I get the scene grpah via pretrained SGG model?

### ISSUE THREAD

**Issue by taotaotao0412** [waxnkw/IETrans-SGG.pytorch#8]
Nice job! I want to get more precious detection result because Faster RCNN with low accurate in some complex scene. I want to get scene graph via image, bbox and label. How should I do. THANKS FOR YOUR HELP!

**Comment by waxnkw** [comment]
Sorry, I do not add the predcls demo.

You can modify one line at [here](https://github.com/waxnkw/IETrans-SGG.pytorch/blob/ddf13cbfeae4e17ee670acf9e210d8c92f93e7d9/maskrcnn_benchmark/data/datasets/visual_genome.py#L156) to add your labels. Note that your provided labels should be aligned with VG's object class ids. 
Then modify the `MODEL.ROI_RELATION_HEAD.USE_GT_OBJECT_LABEL` to `True` in the [VG1800-SGCLS](https://github.com/waxnkw/IETrans-SGG.pytorch/blob/master/cmds/1000/motif/demo_sgcls.sh) or [VG50-SGCLS](https://github.com/waxnkw/IETrans-SGG.pytorch/blob/master/cmds/50/transformer/demo_sgcls.sh).

**Comment by taotaotao0412** [comment]
OK, Thanks a lot
